build.sh

```diff
@@ -11,25 +11,42 @@ RUN_TESTS=${RUN_TESTS:-1}
 VERSION="$(git describe --dirty --always --tags | sed 's/-/./2' | sed 's/-/./2')"
 BUILDDATE="$(date -u '+%Y-%m-%dT%H:%M:%S.%NZ')"
 
+GO_DEP_VERSION="${GO_DEP_VERSION}"
```

Comment:
Suggestion: `${GO_DEP_VERSION:-latest}`

It took me a while to track down where the dep version was actually set. It would be good to put it here w/ the other defaults.

Reply:
Setting it to `latest` breaks the script. It expects either a version tag or empty string (as latest).

Reply:
I guess I can see that here, https://github.com/gluster/gluster-csi-driver/pull/85/files#diff-b6e2d673cd08c78537e77f91e9e2272aR11, at the version check. Not ideal, but :man_shrugging: